libs/payments/cart/src/lib/checkout.service.ts

```diff
@@ -51,15 +53,31 @@ export class CheckoutService {
     private customerManager: CustomerManager,
     private eligibilityService: EligibilityService,
     private invoiceManager: InvoiceManager,
+    private notifierService: NotifierService,
     private paymentMethodManager: PaymentMethodManager,
     private paypalBillingAgreementManager: PaypalBillingAgreementManager,
     private paypalCustomerManager: PaypalCustomerManager,
     private productConfigurationManager: ProductConfigurationManager,
+    private profileClient: ProfileClient,
     private promotionCodeManager: PromotionCodeManager,
     private subscriptionManager: SubscriptionManager,
     @Inject(StatsDService) private statsd: StatsD
   ) {}
 
+  /**
+   * Reload the customer data to reflect a change.
+   */
+  private async customerChanged(uid: string) {
+    await this.profileClient.deleteCache(uid);
+
+    this.notifierService.send({
+      event: 'profileDataChanged',
+      data: {
+        uid,
+      },
+    });
```

Comment:
Here's where I'd suggest extending the Notifier service instead. It should have methods to send each type of event to prevent errors on the caller. e.g. add a new method to the NotiferService called `profileChange` that takes the arguments needed and internally uses send.

Reply:
This doesn't seem to match the use of the event elsewhere, e.g. in the admin server it looks like this:
```typescript
    await this.notifier.send({
      event: 'profileDataChange',
      data: {
        ts: Date.now() / 1000,
        uid,
      },
    });
```